src/extends/PolicySelect/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Select } from '@b-design/ui';
+import { getPolicyList } from '../../api/application';
+import { PoliciesBase } from '../../interface/application';
+import locale from '../../utils/locale';
+import i18n from '../../i18n';
+
+type Props = {
+  onChange: (value: any) => void;
+  policySelectDataSource?: string[];
+  value: any;
+  id: string;
+  disabled: boolean;
+  appName: string;
```

Comment:
We should get the app name from the `uischema` model in the dva, so we should save the app name to the model before rendering the  UISchema component in the application layout.

Reply:
Good idea, let me try it

Reply:
done